data/tutorials/language/0it_00_values_functions.md

```diff
@@ -166,98 +166,97 @@ ha ha
 
 **Note**: As explained in the [Tour of OCaml](/docs/tour-of-ocaml) tutorial, the `unit` type has a single value `()`, which is pronounced "unit."
 
-Above, the pattern does not contain any identifier, meaning no name is defined. The expression is evaluated, the side effect takes place (printing `ha ha` to standard output), no definition is created, and no value is returned. Writing that kind of pseudo-definition only expresses interest in the side effects.
-```ocaml
-# print_endline "ha ha";;
-ha ha
-- : unit = ()
+Above, the pattern does not contain any identifier, meaning no name is defined. The expression is evaluated and the side effect takes place (printing `ha ha` to standard output).
 
-# let _ = print_endline  "ha ha";;
-ha ha
-- : unit = ()
-```
+**Note**: In compiled files to evaluate an expression only for its side effects you must write them after `let () =`.
 
 <!-- user-defined single constructor variant example -->
 <!-- FIXME: create an example nested pattern matching -->
+
+### Pattern Matching on User-defined types
+
 This also works with user-defined types.
 ```ocaml
 # type live_person = int * name;;
 type live_person = int * name
 
-# let age ((years, _) : live_person) = years;;
+# let age (years, { first, last }) = years;;
```

Comment:
I get this error: `Error: Syntax error: '}' expected, the highlighted '{' might be unmatched`